flink-runtime/src/test/java/org/apache/flink/runtime/rpc/akka/AkkaRpcActorTest.java

```diff
@@ -609,6 +611,7 @@ public void onStart() throws Exception {
 
 		public void awaitUntilOnStartCalled() throws InterruptedException {
 			countDownLatch.await();
+			assertThat(getState(), is(State.STARTED));
```

Comment:
Please don't piggy back on existing tests to check for the newly introduced functionality. It is usually a better idea to add targeted tests for the new behaviour.